### PR TITLE
{lang}[intel/2016a] Python 3.5.1 (REVIEW)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/257196b427fcca986582 for a full test report.

File: easybuild/easyconfigs/p/Python/Python-3.5.1-intel-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    ('mpi4py', '2.0.0', {
+        'source_urls': ['http://bitbucket.org/mpi4py/mpi4py/downloads/'],
+    }),
+    ('paycheck', '1.0.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/paycheck/'],
+        'patches': [
+            'paycheck-1.0.2_setup-open-README-utf8.patch',
+        ],
+    }),
+    ('pbr', '0.11.1', {
```
Comment (boegel) [inline easybuild/easyconfigs/p/Python/Python-3.5.1-intel-2016a.eb]: apparently the latest version here is 1.8.1 (and the lockfile below needs it)

The default version you see on PyPI is wrong, cfr. https://pypi.python.org/pypi/pbr/1.8.1 and https://launchpad.net/pbr/+series

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/d7beda4ddee153a3c3fd for a full test report.

**Comment by hajgato** [comment]
@boegel: strange thing that it worked for me. Anyway, I will upgrade to the 1.8.1 version....

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/22ee553b7629e329510f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/ade018a41c2ffe51efa2 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @hajgato!

**Pull request merged**